src/fetcher.ts

```diff
@@ -1596,8 +1596,8 @@ export default class Fetcher implements CallbackifyInterface {
     doc: NamedNode,
     header: string
   ): undefined | string[] {
-    const kb = this.store
-    const requests = kb.each(undefined, this.ns.link('requestedURI'), doc) as Quad_Subject[]
+    const kb = this.store // look for the URI (AS A STRING NOT A NODE) for a stored request
+    const requests = kb.each(undefined, this.ns.link('requestedURI'), doc.uri) as Quad_Subject[]
```

Comment:
I am wondering if we should add uri to tf-types.ts under export interface NamedNode extends Term..  line 30.  This might solve the build issue, looking at how we handled this in solid-ui.

Reply:
index.d.ts in solid-ui has uri as a member of the class NamedNode which extends Node....

Reply:
This was already fixed in the master branch.